workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/ShellActionNode.java

```diff
@@ -34,8 +34,8 @@ The method getXML() returns a string which contains name, Id, next success node(
 
 public class ShellActionNode extends GenericActionNode {
 
-    private static final Logger LOGGER = Logger.getLogger(ShellActionNode.class);
     private ProcessInfo processInfo = new ProcessInfo();
+    private static final String SCRIPT = "script";
     private ActionNode actionNode = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "actionNode" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)